spec/workspace-spec.coffee

```diff
@@ -76,7 +76,7 @@ describe "Workspace", ->
           expect(editor4.getCursorScreenPosition()).toEqual [2, 4]
 
           expect(atom.workspace.getActiveTextEditor().getPath()).toBe editor3.getPath()
-          expect(document.title).toBe "#{path.basename(editor3.getPath())} - #{atom.project.getPaths()[0]} - Atom"
+          expect(document.title).toMatch ///^#{path.basename(editor3.getPath())}\ \u2014\ #{atom.project.getPaths()[0]}///
```

Comment:
@mnquintana It was pointed out to me by @50Wliu that string interpolation doesn't work in standard regexes.